src/devices/Bmxx80/Bme680.cs

```diff
@@ -450,7 +464,11 @@ protected override void SetDefaultConfiguration()
             FilterMode = Bme680FilteringMode.C0;
 
             _bme680Calibration = (Bme680CalibrationData)_calibrationData;
-            TryReadTemperature(out var temp);
+            if (!TryReadTemperature(out var temp))
+            {
+                temp = _ambientTemperatureDefault;
```

Comment:
Should we do a Debug.WriteLine here somewhere to log that there was an issue reading the temperature?

Reply:
Of course, we could do that (even inside the TryReadTemperature method), but we generally don't do this. In several other PRs/tickets we saw that a generic logging framework for the whole project would probably be helpful (i.e. an interface where one could attach a console, or an NLog writer or something). I'd rather go that way (in a separate PR).